test/extended/builds/start_build.go

```diff
@@ -160,7 +160,7 @@ var _ = g.Describe("builds: parallel: oc start-build", func() {
 
 			g.By("getting the build name")
 			var buildName string
-			wait.Poll(time.Duration(100*time.Millisecond), time.Duration(60*time.Second), func() (bool, error) {
+			wait.Poll(time.Duration(100*time.Millisecond), time.Duration(time.Minute), func() (bool, error) {
```

Comment:
@rhcarvalho I've chaned only one what you've pointed and leave the rest (like `time.Duration(100*time.Millisecond`).